common/config/rush/pnpmfile.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  hooks: {
+    readPackage
+  }
+};
+
+function readPackage (pkg) {
+  if (pkg.name === 'jest-runtime') {
+    pkg.dependencies['slash'] = '^1.0.0';
+  }
+  return pkg;
+}
```

Comment:
What is this? Does every project that uses PNPM need this file?

Reply:
This is a special override for "bad packages" (e.g. jest-runtime didn't add slash as a dependency). There is a long discussion about this file here [https://github.com/pnpm/pnpm/issues/949](https://github.com/pnpm/pnpm/issues/949)

Goal here is to get WBT working with PNPM before making additional changes to pnpm

---
In reply to: [159794827](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159794827) [](ancestors = 159794827)

Reply:
I tested and we don't need this since newer version of jest-runtime arent missing the dependency

---
In reply to: [159795139](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159795139) [](ancestors = 159795139,159794827)